test/unit/test_visual_recognition_v4.py

```diff
@@ -12,824 +13,1200 @@
-                },
-                "image_id": "image_id",
-                "updated": "2000-01-23T04:56:07.000+00:00",
-                "errors": {
-                    "code":
-                        "invalid_field",
-                    "message":
-                        "The date provided for `version` is not valid. Specify dates in `YYYY-MM-DD` format.",
-                    "more_info":
-                        "https://cloud.ibm.com/apidocs/visual-recognition-v4#versioning",
-                    "target": {
-                        "type": "parameter",
-                        "name": "version"
-                    }
-                },
-                "dimensions": {
-                    "width": 6,
-                    "height": 0
-                }
-            }, {
-                "training_data": {
-                    "objects": [{
-                        "location": {
-                            "top": 1,
-                            "left": 5,
-                            "width": 5,
-                            "height": 2
-                        },
-                        "object": "object"
-                    }, {
-                        "location": {
-                            "top": 1,
-                            "left": 5,
-                            "width": 5,
-                            "height": 2
-                        },
-                        "object": "object"
-                    }]
-                },
-                "created": "2000-01-23T04:56:07.000+00:00",
-                "source": {
-                    "archive_filename": "archive_filename",
-                    "filename": "filename",
-                    "type": "file",
-                    "resolved_url": "resolved_url",
-                    "source_url": "source_url"
-                },
-                "image_id": "image_id",
-                "updated": "2000-01-23T04:56:07.000+00:00",
-                "errors": {
-                    "code":
-                        "invalid_field",
-                    "message":
-                        "The date provided for `version` is not valid. Specify dates in `YYYY-MM-DD` format.",
-                    "more_info":
-                        "https://cloud.ibm.com/apidocs/visual-recognition-v4#versioning",
-                    "target": {
-                        "type": "parameter",
-                        "name": "version"
-                    }
-                },
-                "dimensions": {
-                    "width": 6,
-                    "height": 0
-                }
-            }],
-            "trace":
-                "trace",
-            "warnings": [{
-                "code": "invalid_field",
-                "more_info": "more_info",
-                "message": "message"
-            }, {
-                "code": "invalid_field",
-                "more_info": "more_info",
-                "message": "message"
-            }]
-        }
```

Comment:
Were all of these responses moved to an external file?

Reply:
For `vr4` used the autogenerated file